app/backend/src/couchers/models.py

```diff
@@ -1058,3 +1068,14 @@ class Reply(Base):
     deleted = Column(DateTime(timezone=True), nullable=True)
 
     comment = relationship("Comment", backref="replies")
+
+
+class LanguageAbilities(Base):
```

Comment:
The class name should be singular, i.e. `LanguageAbility`, also, it would be good to move this to be, say, directly under the `User` model

Reply:
but it has multiple language abilities.

Reply:
Each of these objects represents a single language ability (the table should be called `language_abilities` though), it's the way you use it

Reply:
Got it, I'll change it.